src/spaces/a98rgb.js

```diff
@@ -3,13 +3,9 @@ import A98Linear from "./a98rgb-linear.js";
 
 export default new RGBColorSpace({
 	id: "a98rgb",
+	cssId: "a98-rgb",
```

Comment:
Why do we need to specify this? It should default to `this.id`. This was meant for things like `p3` -> `display-p3`. I can also see having it for dashed-ident color spaces. But for the default ones, why?

Reply:
@LeaVerou Correct, this is only for cases where the `cssId` is different than the `id` -- but that's the case here too... `a98rgb` is different than `a98-rgb`. Am I misunderstanding your question?

Reply:
This doesn't change anything functionally, but uses the shorter/clearer `cssId: "a98-rgb"` instead of `formats: { color: { id: "a98-rgb" } }`.

Reply:
Should we consider this a bug in the `id` and change the id to `"a98-rgb"`, remove the `cssId` and add an alias for `a98rgb`. If not then we should add an alias of `"a98-rgb"` to the color space.

Reply:
Oh I see, there's an extra hyphen in the CSS version. I totally missed that and thought you were repeating the same id (which should not be needed)

Reply:
> Should we consider this a bug in the id and change the id to "a98-rgb", remove the cssId and add an alias for a98rgb. If not then we should add an alias of "a98-rgb" to the color space.

Since the color space ids are also used as a accessors, it would provide poor ergonomics to have ids with hyphens, so I think this is fine.